src/vault.py

```diff
@@ -12,6 +12,15 @@
 
 logger = logging.getLogger(__name__)
 
+KV_MOUNT_HCL = """
+path "{mount}/*" {{
+  capabilities = ["create", "read", "update", "delete", "list"]
+}}
+path "sys/internal/ui/mounts/{mount}" {{
+  capabilities = ["read"]
+}}
+"""
+
```

Comment:
This could have its own .hcl file which will help readability/parsing it. (I know this one is quite small but still...)